package.json

```diff
@@ -28,14 +28,15 @@
   ],
   "dependencies": {
     "@types/fancy-log": "1.3.0",
-    "chalk": "2.3.1",
+    "ansi-colors": "^1.0.1",
```

Comment:
I prefer to lock down versions, e.g. `"1.0.1"`, but I can fix this.

Reply:
Didn't even notice the lack of the caret. For a build tool, I can understand the preference to lock down the dependencies.